content/en/docs/setup/production-environment/tools/kubeadm/install-kubeadm.md

```diff
@@ -199,6 +199,7 @@ enabled=1
 gpgcheck=1
 repo_gpgcheck=1
 gpgkey=https://packages.cloud.google.com/yum/doc/yum-key.gpg https://packages.cloud.google.com/yum/doc/rpm-package-key.gpg
+exclude=kubelet kubeadm kubectl
```

Comment:
just to clarify, `exclude` here would skip updates of the listed packages during distro upgrade?
and then the user must always pass `--disableexcludes` on `yum install`?

looks like our upgrade guide already has the flag everywhere:
https://kubernetes.io/docs/tasks/administer-cluster/kubeadm/kubeadm-upgrade/

Reply:
@neolit123 yes, that's my understanding. In fact, just below this section there is a `yum install --disableexcludes=kubernetes` which is what made me notice that there wasn't an exclusion in place to exclude.

Reply:
thanks for confirming.
/lgtm